Update field limits and image validation

driftingly opened the pull request:
This PR updates the field limits and adds additional image validation.

nikhiltri reviewed (approved):
Some of the hints don't match the validation rules. But outside of that looks good! Marking this as approve, go ahead and consider this a thumbs up to merge once that small detail is fixed. 

<img width="893" alt="Screenshot 2024-05-01 at 4 45 44 PM" src="https://github.com/art-institute-of-chicago/journeymaker-cms/assets/1613175/aea67f95-4066-4fdb-bc0e-4c0f0043caa6">

zachgarwood reviewed (approved):
Makes sense

driftingly commented:
> Some of the hints don't match the validation rules. But outside of that looks good! Marking this as approve, go ahead and consider this a thumbs up to merge once that small detail is fixed.
> 
> <img alt="Screenshot 2024-05-01 at 4 45 44 PM" width="893" src="https://private-user-images.githubusercontent.com/1613175/327257117-aea67f95-4066-4fdb-bc0e-4c0f0043caa6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ2MDQxODUsIm5iZiI6MTcxNDYwMzg4NSwicGF0aCI6Ii8xNjEzMTc1LzMyNzI1NzExNy1hZWE2N2Y5NS00MDY2LTRmZGItYmMwZS00YzBmMDA0M2NhYTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDFUMjI1MTI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzEzODBmMGY0NDI5MzRiMTU2NjQ3MzhkZmI2YTIwMjZmMGIyNWMwM2E2MjdiMGZkMDVjN2EyZjY3MjkxOWI2MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lBYclfXbJQ4lzRWy9364kJ8NNpUJ4G6BOSzR0_EKVf8">

Thanks @nikhiltri I was showing Jeanne how we could add notes and didn't mean for that to make it through. I think it's worth including so I updated and added them to the other fields.
